sql/core/src/test/scala/org/apache/spark/sql/MergeAsOfSuite.scala

```diff
@@ -498,4 +500,136 @@ class MergeAsOfSuite extends QueryTest with SharedSQLContext{
         Row(new Timestamp(105), 20, "c", null)
       ))
   }
-}
+
+  test("generated intervalized test") {
```

Comment:
Can you add more tests with different params configuration? i.e., tolerance, bias, etc?